__tests__/UserCaption-test.js

```diff
@@ -1,12 +1,186 @@
 import 'react-native';
 import React from 'react';
 import UserCaption from '@screens/app/Components/UserCaption/UserCaption';
+import checkPropTypes from 'check-prop-types';
+var assert = require('assert');
```

Comment:
Where is this being pulled from?

Reply:
What do you mean by "this"? assert is a native js function and check-prop-types is a dependency I have added. You can find the link here : https://www.npmjs.com/package/check-prop-types